lib/board.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class Board
+  attr_accessor :board
+
+  def initialize
+    @board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+    @winner_points = [0, 1, 2, 3, 4, 5, 6, 7, 8, 0, 3, 6, 1, 4, 7, 2, 5, 8, 0, 4, 8, 6, 4, 2]
+  end
+
+  def update_board(num, char)
+    @board[num - 1] = char
+  end
+
+  def check_board
```

Comment:
Let's have a more meaningful naming convention for our methods. For instance, I believe this method here checks if the board is already full right? So probably I'd name this method something like: `full?`. So when you call it outside it will be `board.full?`